tests/hazmat/primitives/test_decrepit.py

```diff
@@ -0,0 +1,340 @@
+# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
This filename doesn't line up with the name of the module with teh code?

Reply:
should we do `hazmat/primitives/decrepit/test_algorithms.py`?

Reply:
yeah